docs/user/dql/functions.rst

```diff
@@ -2436,6 +2436,28 @@ Example::
     | 2020-08-26 13:49:00                | 2020-08-27 02:04:42                                  |
     +------------------------------------+------------------------------------------------------+
 
+TIMESTAMPADD
+------------
+
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a DATETIME value based on a passed in DATE/DATETIME/TIME/TIMESTAMP/STRING argument and an INTERVAL and INTEGER argument which determine the amount of time to be added.
+If an third argument of type STRING is provided, it must be formatted as a valid DATETIME. If only a TIME is provided, a DATETIME is still returned with the DATE portion filled in using the current date.
+
+Argument type: INTERVAL, INTEGER, DATE/DATETIME/TIME/TIMESTAMP/STRING
+INTERVAL must be one of the following tokens: [MICROSECOND, SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, QUARTER, YEAR]
+
+Examples::
+
+    os> SELECT TIMESTAMPADD(MINUTE, 1, '2003-01-02 00:00:00');
```

Comment:
Maybe: more [complex] example(s)

Reply:
negative add would be good

Reply:
Added in https://github.com/Bit-Quill/opensearch-project-sql/pull/246/commits/77226baed22abe00079b735df28ff26f2cc7547e